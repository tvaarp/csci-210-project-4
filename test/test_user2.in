sendmsg user1 nice to meet you!
sendmsg user1 bye
