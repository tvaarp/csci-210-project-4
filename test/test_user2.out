Incoming message from user1: hello there
Incoming message from user1: see you later
Exiting....
